htcrystalball/collect.py

```diff
@@ -8,14 +8,15 @@
 
 
 def collect_slots(filename: Union[str, None] = None) -> dict:
-    """Get the condor config and creates a dict."""
-    projection = ["SlotType", "UtsnameNodename", "TotalSlotCpus", "TotalSlotDisk", "TotalSlotMemory", "TotalSlots",
+    """Gets the condor config and creates a dict."""
+    query_data = ["SlotType", "UtsnameNodename", "TotalSlotCpus", "TotalSlotDisk", "TotalSlotMemory", "TotalSlots",
                   "TotalSlotGPUs"]
     unique_slots = {}
 
     if filename is None:
         coll = htcondor.Collector()
-        content = coll.query(htcondor.AdTypes.Startd, constraint='SlotType != "Dynamic"', projection=projection)
+        # Ignore Slots that are Dynamic as they are dynamically assigned to a job and cannot be predicted
```

Comment:
I think this fails to explain the ephemeral nature of dynamic slots.

> Ignore dynamic slots, which are the ephemeral children of partitionable slots, and thus noise. 
> Partitionable slot definitions remain unaltered by the process of dynamic slot creation.

Reply:
Alright, I just updated the comment